tests/write_plugin_graph/screen-output

```diff
@@ -0,0 +1,28 @@
+-----------------------------------------------------------------------------
+-----------------------------------------------------------------------------
+
+Loading shared library <./libwrite_plugin_graph.so>
+
+Number of active cells: 16 (on 3 levels)
+Number of degrees of freedom: 268 (162+25+81)
+
+found digraph Plugins
+found label="TestMaterial"
```

Comment:
Ah, clever -- I was wondering why you are creating the test material model. Nice idea!